js/mandelbrot.js

```diff
@@ -1,17 +1,17 @@
-const WIDTH = 1200, HEIGHT = 800;
+const WIDTH  = 1200;
+const HEIGHT = 800;
 
-const colour = (iteration, offset, scale) => {
-  iteration = ((iteration * scale) + offset) % 1024;
+function colour(iteration, offset, scale) {
+  iteration = (iteration * scale + offset) & 1023;
   if (iteration < 256) {
     return iteration;
   } else if (iteration < 512) {
-    return 255 - (iteration - 255);
-  } else {
-    return 0;
+    return 255 - ((iteration & 255) - 255);
```

Comment:
Ah - with respect to https://github.com/ColinEberhardt/wasm-mandelbrot/pull/3/files#r200177666 - the color scale has changed here

Reply:
Yap, already fixed)